### PR TITLE
FAQ update as per 0.9.0

### PR DESCRIPTION
**Pull request by joseph619**

### CODE REVIEW

File: src/help/faq/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Read more [here](/help/guides/settings/advanced/#refresh-library-metadata)
 Every now and then you find a manga that is translated by more than one group, because of this you get multiple releases for every chapter in the manga. To fix this the easiest way is to change sorting to **Total chapters**. You won't see any changes but, when you start reading you'll automatically skip the duplicate releases.
```
Comment (arkon) [inline src/help/faq/README.md:30]: Could maybe mention just marking them as read and turning on the skip filtered chapters setting? Can be a separate PR.

File: src/help/faq/README.md [hunk 2]
```diff
@@ -81,27 +81,24 @@ If you don't see a **Tachiyomi** folder on your device, try downloading a chapte
 
 ### Application Updates
 #### How do I enable automatic updates?
-The **Tachiyomi** app now has in-app updating for both stable and preview versions. It happens automatically in the background if you got **[Check for updates](/help/guides/settings/about/#check-for-updates)** enabled.
-::: guide
-If you go to <Navigation item="settings"/> → <Navigation item="old_about"/> you will find the **[Check for updates](/help/guides/settings/about/#check-for-updates)** toggle, you can also press **[Version](/help/guides/settings/about/#version)** to force check for updates.
-:::
+From version **0.9.0** and onwards, Tachiyomi has Auto-update for the app enabled by default.
```
Comment (arkon) [inline src/help/faq/README.md:84]:
```diff
- Auto-update
+ automatic updates
```

Comment (arkon) [reply src/help/faq/README.md:84]: Should also retain the mention about Preview, since some people still assume it/dev doesn't have it.

File: src/help/faq/README.md [hunk 3]
```diff
@@ -81,27 +81,24 @@ If you don't see a **Tachiyomi** folder on your device, try downloading a chapte
...
-The **Tachiyomi** app now has in-app updating for both stable and preview versions. It happens automatically in the background if you got **[Check for updates](/help/guides/settings/about/#check-for-updates)** enabled.
-::: guide
-If you go to <Navigation item="settings"/> → <Navigation item="old_about"/> you will find the **[Check for updates](/help/guides/settings/about/#check-for-updates)** toggle, you can also press **[Version](/help/guides/settings/about/#version)** to force check for updates.
-:::
+From version **0.9.0** and onwards, Tachiyomi has Auto-update for the app enabled by default. 
 
 ### User Interface
 #### Can I change the app language?
-**Tachiyomi** is as of writing available in 35 different languages. By default, the app follows the language used on your device.
+**Tachiyomi** is as of writing available in 40 different languages. By default, the app follows the language used on your device.
```
Comment (arkon) [inline src/help/faq/README.md:88]:
```diff
- **Tachiyomi** is as of writing available in 40 different languages.
+ **Tachiyomi** is translated in 40+ different languages.
```

File: src/help/faq/README.md [hunk 4]
```diff
@@ -115,26 +112,33 @@ There is currently no iOS version and there are no plans for one and since iOS a
 **Tachiyomi** is still in beta and some design changes or unexpected errors may lead to data loss. While we try for them not to happen, sometimes it is unavoidable. To avoid this in the future, you can use the automatic backup feature.
 
 ::: aside
-Learn how to backup [here](/help/guides/settings/backup/)
+Learn how to create Auto-backup [here](/help/guides/creating-backups/#turning-on-auto-backups/)
```
Comment (arkon) [inline src/help/faq/README.md:115]:
```diff
- Auto-backup
+ automatic backups
```

File: src/help/faq/README.md [hunk 5]
```diff
@@ -115,26 +112,33 @@ There is currently no iOS version and there are no plans for one and since iOS a
...
-- Other popular sources: **Mangahere**, **Mangapark**, **Manhwa18.net** (in **FMReader** extension)
+- Webtoons and Manhwa/Manhua: **Webtoons**, **Toonily** (found in **Madara** extension), **Mangazuki.me** (in **Madara** extension), **IsekaiScan.com** (also in **Madara** extension)
+- Western comics: **ReadComicOnline** and **ComicExtra**
+- Yaoi, Yuri, Shounen Ai and Shoujo Ai: **Mangahasu**, **Dynasty** (for Yuri), **Lily Manga** (for Yuri, in **Madara** extension) **MyReadingManga** (for Yaoi)
+- Other popular sources: **Mangahere**, **Manhwa18.net** (in **FMReader** extension), **ReadManhwa**, **MangaOwl**
 
 Most of the scanlator sources are part of **Multi-source** extensions, which contain multiple sources. For example, extensions like **Genkan**, **Madara**, **FoolSlide** and **ComiCake** which contain sources which host directly from the scanlators' own websites.
 
+#### What are scanlator sources?
+Manga that are not officially licensed are translated by scanlators, who either publish manga on their own website or **MangaDex**. Scanlator sources always provide the fastest manga updates and have highest scan quality.
```
Comment (arkon) [inline src/help/faq/README.md:138]: Can make "scanlators" a link to https://en.wikipedia.org/wiki/Scanlation

File: src/help/faq/README.md [hunk 6]
```diff
@@ -188,11 +192,11 @@ If you need more help regarding this, read [this](https://www.theandroidsoul.com
 Extensions for **Tachiyomi** are technically installed the same as normal apps. Either you uninstall them through the app section on your phones settings or you go can do it in the app.
 
 ::: guide
-To uninstall an extension from within **Tachiyomi**, go to <Navigation item="old_extensions"/> then tap **Uninstall** on the extension you want to remove.
+To uninstall an extension from within **Tachiyomi**, go to <Navigation item="browse"/> → **EXTENSIONS** then tap **Uninstall** on the extension you want to remove.
 :::
 
 #### Why don't I have extensions or any sources in the catalogue?
-- **Tachiyomi** no longer have pre-installed extensions, this means that you will need to install extensions you want to use from the <Navigation item="old_extensions"/> menu.
+- **Tachiyomi** no longer have pre-installed extensions, this means that you will need to install extensions you want to use from the <Navigation item="browse"/> → **EXTENSIONS** menu.
```
Comment (arkon) [inline src/help/faq/README.md:199]: We should just remove mentions about pre-installed extensions at this point

File: src/help/faq/README.md [hunk 7]
```diff
@@ -208,29 +212,16 @@ To uninstall an extension from within **Tachiyomi**, go to <Navigation item="old
 10. WieManga
 :::
 
-::: warning
-Be aware that **Batoto** is deprecated so you will need to migrate from that source to others.
-::: aside
-Read more about Batoto shutting down [here](https://goboiano.com/manga-scanlation-site-batoto-to-close-down/)
-:::
-
-#### Why are my extensions not showing up in Catalogues?
```
Comment (arkon) [inline src/help/faq/README.md:217]: Should rework this section for language filtering, which seems to confuse some people.

File: src/help/faq/README.md (hunk 1 above)
Comment (joseph619) [reply src/help/faq/README.md:30]: Maybe in next PR yeah.

File: src/help/faq/README.md (hunk 7 above)
Comment (joseph619) [reply src/help/faq/README.md:217]: Added language filtering FAQ

**Comment by Soitora** [comment]
Man you can't just add navigation items without adding them to the Navigation.vue first, that's what makes the check failing

**Comment by joseph619** [comment]
lol I thought filter button already existed, my bad

**Comment by Soitora** [comment]
It does, but you added a Toggle one

**Comment by joseph619** [comment]
I removed the toggle, it's not really needed.
